CLI/1. User Guides/2. 登入.md

```diff
@@ -0,0 +1,86 @@
+---
+title: 登入 TWCC CLI | en
+tags: Guide, CLI, EN
+GA: UA-155999456-1
+---
+
+{%hackmd @docsharedstyle/twccheader-en %}
+{%hackmd @docsharedstyle/default %}
+
+# Log in to TWCC CLI
+
+After installation, please prepare **Project ID**, **API key (login certificate)**, and then log in to TWCC CLI to start.

```

Comment:
prepare > prepare your

log in to TWCC CLI to start. > sign in with TWCC CLI to get started.